packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js

```diff
@@ -74,4 +75,19 @@ describe('BlocksEditor', () => {
 
     expect(screen.getByRole('img')).toBeInTheDocument();
   });
+
+  it.skip('when there is no selection, selecting modifiers should work at the end of the editor', async () => {
+    setup({ value: null });
+
+    const boldButton = screen.getByLabelText(/bold/i);
+    await userEvent.click(boldButton);
+
+    const editorContainer = screen.getByTestId('slate-editor');
+    // domRange.getBoundingClientRect = jest.fn(); TODO: mock getBoundingClientRect on a selection element
+    userEvent.type(editorContainer, 'bold text here');
+
+    expect(screen.getByText('bold text here').parentElement).toHaveStyle({
+      'font-weight': 600,
+    });
+  });
```

Comment:
@remidej @simotae14 On initial click on modifiers I am setting selection to EOL, to test this behaviour I need to type some text to test whether modifiers are working or not. 
For that I need Editor in scope to test it and so I added a test case here. But I am getting exception `domRange.getBoundingClientRect not a function` as we do not have real dom to test this. I need help on how can we make instance of Range and make it work or if you have any other suggestion on how to type in editor to test modifiers work or not.